Source/LibationFileManager/Configuration.PersistentSettings.cs

```diff
@@ -3,336 +3,250 @@
+		public bool MergeOpeningAndEndCredits { get => GetNonString<bool>(); set => SetNonString(value); }
+
+		[Description("Strip \"(Unabridged)\" from audiobook metadata tags")]
+		public bool StripUnabridged { get => GetNonString<bool>(); set => SetNonString(value); }
+
+		[Description("Strip audible branding from the start and end of audiobooks.\r\n(e.g. \"This is Audible\")")]
+		public bool StripAudibleBrandAudio { get => GetNonString<bool>(); set => SetNonString(value); }
+
+		[Description("Decrypt to lossy format?")]
+		public bool DecryptToLossy { get => GetNonString<bool>(); set => SetNonString(value); }
+
+		[Description("Lame encoder target. true = Bitrate, false = Quality")]
+		public bool LameTargetBitrate { get => GetNonString<bool>(); set => SetNonString(value); }
+
+		[Description("Lame encoder downsamples to mono")]
+		public bool LameDownsampleMono {  get => GetNonString<bool>(); set => SetNonString(value); }
+
+		[Description("Lame target bitrate [16,320]")]
+		public int LameBitrate {  get => GetNonString<int>(); set => SetNonString(value); }
+
+		[Description("Restrict encoder to constant bitrate?")]
+		public bool LameConstantBitrate { get => GetNonString<bool>(); set => SetNonString(value); }
+
+		[Description("Match the source bitrate?")]
+		public bool LameMatchSourceBR { get => GetNonString<bool>(); set => SetNonString(value); }
+
+		[Description("Lame target VBR quality [10,100]")]
+		public int LameVBRQuality { get => GetNonString<int>(); set => SetNonString(value); }
+
+		[Description("A Dictionary of GridView data property names and bool indicating its column's visibility in ProductsGrid")]
+		public Dictionary<string, bool> GridColumnsVisibilities { get => GetNonString<EquatableDictionary<string, bool>>().Clone(); set => SetNonString(value); }
+
+		[Description("A Dictionary of GridView data property names and int indicating its column's display index in ProductsGrid")]
+		public Dictionary<string, int> GridColumnsDisplayIndices { get => GetNonString<EquatableDictionary<string, int>>().Clone(); set => SetNonString(value); }
+
+		[Description("A Dictionary of GridView data property names and int indicating its column's width in ProductsGrid")]
+		public Dictionary<string, int> GridColumnsWidths { get => GetNonString<EquatableDictionary<string, int>>().Clone(); set => SetNonString(value); }
+
+		[Description("Save cover image alongside audiobook?")]
+		public bool DownloadCoverArt { get => GetNonString<bool>(); set => SetNonString(value); }
+
+		[Description("Download clips and bookmarks?")]
+		public bool DownloadClipsBookmarks { get => GetNonString<bool>(); set => SetNonString(value); }
+		
+		[Description("File format to save clips and bookmarks")]
+		public ClipBookmarkFormat ClipsBookmarksFileFormat { get => GetNonString<ClipBookmarkFormat>(); set => SetNonString(value); }
+
+		[JsonConverter(typeof(StringEnumConverter))]
+		public enum ClipBookmarkFormat
```

Comment:
With this property, enums will be serialized as strings instead of as numbers.

Reply:
Good idea. How will this work retro actively? ie -- with existing enum settings (assuming they exist)

Reply:
Works seamlessly with existing settings files. The comment right below this one shows how enums are deserialized.